spring-cloud-azure-autoconfigure/src/main/java/com/microsoft/azure/spring/cloud/autoconfigure/storage/AzureStorageAutoConfiguration.java

```diff
@@ -35,17 +38,17 @@
 @ConditionalOnClass(CloudStorageAccount.class)
 @ConditionalOnProperty(name = "spring.cloud.azure.storage.account")
 @EnableConfigurationProperties(AzureStorageProperties.class)
+@Import(AzureStorageProtocolResolver.class)
 public class AzureStorageAutoConfiguration {
```

Comment:
Shouldn't it be AzureStorageAccountAutoConfiguration since it wraps storage account object here.

Reply:
`Storage` in this class name is more like a general concept, rather than a specific thing. Although currently we only provide a bean of type `storageAccount`, we might provide more storage related beans in the future.